activerecord/lib/active_record/connection_adapters/sqlite3/schema_dumper.rb

```diff
@@ -0,0 +1,13 @@
+module ActiveRecord
+  module ConnectionAdapters
+    module SQLite3
+      module ColumnDumper
+        private
+
+          def default_primary_key?(column)
+            schema_type(column) == :integer
+          end
```

Comment:
@kamipo That's correct. Since the default has changed, other adapters can either choose to join us in `bigint` land (by doing nothing), or can choose to stay in `int` land by overriding.